python/cugraph/structure/number_map.py

```diff
@@ -326,7 +327,7 @@ def compute(self):
                 #  Compute base address for each partition
                 #
                 counts = numbering_map.map_partitions(
-                    lambda df: df.groupby("partition").count()
+                    lambda df: df.groupby("partition", sort=True).count()
```

Comment:
This shouldn't require a `sort=True`.  We're only after a count, order doesn't matter.